x/shared/keeper/msg_server_update_param_test.go

```diff
@@ -94,7 +102,11 @@ func TestMsgUpdateParam_UpdateClaimWindowCloseOffsetBlocks(t *testing.T) {
 	require.Equal(t, uint64(expectedClaimWindowCloseOffestBlocks), res.Params.ClaimWindowCloseOffsetBlocks)
 
 	// Ensure the other parameters are unchanged
-	require.Equal(t, defaultParams.NumBlocksPerSession, res.Params.NumBlocksPerSession)
+	require.Equal(t, defaultParams.GetNumBlocksPerSession(), res.Params.GetNumBlocksPerSession())
```

Comment:
Can you think of a way to add a helper that does something like `CompareAgainsDefaultsExcept`. It might require reflections, but think there's value here that others can reuse.